### ISSUE TITLE
Okko does not download

### ISSUE THREAD

**Issue by ST02-droid** [vitalygashkov/streamyx#56]
### chrome_debug.log
```[1215/132247.300:ERROR:cert_issuer_source_aia.cc(34)] Error parsing cert retrieved from AIA (as DER):
ERROR: Couldn't read tbsCertificate as SEQUENCE
ERROR: Failed parsing Certificate

[1215/132247.569:INFO:CONSOLE(24)] "stating client 3.257.27039 master", source: https://okko.tv/assets/js/847abcf6.js (24)
[1215/132247.570:INFO:CONSOLE(24)] "response cache [object Object]", source: https://okko.tv/assets/js/847abcf6.js (24)
[1215/132247.570:INFO:CONSOLE(24)] "session cache [object Object]", source: https://okko.tv/assets/js/847abcf6.js (24)
[1215/132247.585:INFO:CONSOLE(24)] "location /serial/volshebnyjj-uchastok ", source: https://okko.tv/assets/js/847abcf6.js (24)
[1215/132247.661:INFO:CONSOLE(24)] "rendering page Волшебный участок (сериал, 2023, 1 сезон) смотреть онлайн в хорошем качестве HD (720) / Full HD (1080)", source: https://okko.tv/assets/js/847abcf6.js (24)
[1215/132247.708:ERROR:cert_verify_proc_builtin.cc(705)] CertVerifyProcBuiltin for tagmanager.datago.ru failed:
----- Certificate i=0 (CN=tagmanager.datago.ru) -----
ERROR: Time is after notAfter


[1215/132247.708:ERROR:ssl_client_socket_impl.cc(975)] handshake failed; returned -1, SSL error code 1, net_error -201
[1215/132248.140:INFO:CONSOLE(1)] "start timeout skipped", source: https://okko.tv/assets/js/847abcf6.js (1)
[1215/132248.141:INFO:CONSOLE(1)] "[PlayerFactory]: OkkoPlayer script loaded", source: https://okko.tv/assets/js/847abcf6.js (1)
[1215/132248.197:INFO:CONSOLE(1)] "screenapi https://ctx.playfamily.ru/screenapi/v1/noauth/uiscreeninfo/web/1?sid=SNXKYOAVFn2YMlmh5BywTT__Mi-58gFCT9ZMkmvFH-8Wprat2GSO4Pf4-NwNstmA7_QlPKACtK_yAC_AH8jY2WxurzZ_18S_JCJ9RbypGxk~-10553 status 0", source: https://okko.tv/assets/js/847abcf6.js (1)
[1215/132248.205:INFO:CONSOLE(1)] "start timeout skipped", source: https://okko.tv/assets/js/847abcf6.js (1)
[1215/132248.209:INFO:CONSOLE(1)] "start timeout skipped", source: https://okko.tv/assets/js/847abcf6.js (1)
[1215/132248.489:ERROR:cert_issuer_source_aia.cc(34)] Error parsing cert retrieved from AIA (as DER):
ERROR: Failed parsing Certificate SEQUENCE
ERROR: Failed parsing Certificate

[1215/132248.540:WARNING:spdy_session.cc(2976)] Received HEADERS for invalid stream 3
[1215/132248.542:INFO:CONSOLE(1)] "SW installed and waiting activation", source: https://okko.tv/sw.js (1)
[1215/132248.543:WARNING:spdy_session.cc(2976)] Received HEADERS for invalid stream 7
[1215/132248.548:INFO:CONSOLE(1)] "SW activated", source: https://okko.tv/sw.js (1)
[1215/132248.548:ERROR:cert_issuer_source_aia.cc(34)] Error parsing cert retrieved from AIA (as DER):
ERROR: Failed parsing Certificate SEQUENCE
ERROR: Failed parsing Certificate

[1215/132248.548:ERROR:cert_verify_proc_builtin.cc(705)] CertVerifyProcBuiltin for cms-res.online.sberbank.ru failed:
----- Certificate i=2 (CN=SberCA Root Ext,O=Sberbank of Russia,C=RU) -----
ERROR: No matching issuer found


[1215/132248.549:ERROR:ssl_client_socket_impl.cc(975)] handshake failed; returned -1, SSL error code 1, net_error -202
[1215/132248.551:WARNING:spdy_session.cc(2976)] Received HEADERS for invalid stream 9
[1215/132248.573:WARNING:spdy_session.cc(2976)] Received HEADERS for invalid stream 9
```
![изображение](https://github.com/vitalygashkov/streamyx/assets/77666552/7d71b287-b268-46c4-b200-e5ac19a9d84a)

**Comment by DenSBK** [comment]
Try to do this:
streamyx-win-x64 -s 1 -e 1 https://okko.tv/serial/xxxxxxx
Debe especificar la temporada y el número de episodio o más, por ejemplo, - e 1-20

**Comment by vitalygashkov** [comment]
> Try to do this: streamyx-win-x64 -s 1 -e 1 https://okko.tv/serial/xxxxxxx It is necessary to specify the season and episode number or several, for example, -e 1-20

These arguments are optional. If you don't specify them, the whole series will be downloaded. You can also specify a direct link to the episode like https://okko.tv/***/season/1/episode/1.

**Comment by vitalygashkov** [comment]
Fixed in #57
